README.rst

```diff
@@ -171,6 +172,7 @@ Most of the API is available:
     client.studies.export
     client.studies.export_by_username
     client.studies.import_pgn
+    client.studies.get_from_user
```

Comment:
get_metadatas_by_username is more descriptive

Reply:
Ah cool, I thought of something on similar lines but decided it was getting too lengthy